### DISCUSSION TITLE
How to call an external-command with multiple arguments stored in a list

### DISCUSSION THREAD

**Discussion by emailstorbala** [nushell/nushell#11585] [Q&A] [answered]
I want to call few commands like tar in the below fashion.

`run-external "tar" "-czf" $target_file_name $inp_file_and_dir_list`

here,
target_file_name -> a string variable
inp_file_and_dir_list -> a list<string> - ["test1.sh" "test2.sh"]

Here I want to compress/tar a list of files and directories.

**Comment by amtoine** [comment]
is there something not working? it looks pretty ok to me

**Reply by emailstorbala** [reply, marked as answer]
Fixed it!
Its not working as such. Facing the error "Spreading lists automatically when calling external commands is deprecated and will be removed in 0.91."

`run-external "tar" "-czf" $target_file_name ...$inp_list`
